build.sbt

```
@@ -8,28 +8,20 @@ name := """fs2-rabbit-root"""
 organization in ThisBuild := "dev.profunktor"
 
 scalaVersion in ThisBuild := "2.12.9"
-crossScalaVersions in ThisBuild := Seq("2.11.12", "2.12.9", "2.13.0")
-
-// makes `tut` fail :( -> https://github.com/tpolecat/tut/issues/255
-//scalaVersion in ThisBuild := "2.12.8" // needed for metals
+crossScalaVersions in ThisBuild := Seq("2.12.9", "2.13.0")
```

Do we want to set this to 2.12.10?
```suggestion
crossScalaVersions in ThisBuild := Seq("2.12.10", "2.13.0")
```

---

Yup, I just made the change in my local branch! :smile: I was making sure everything compiles before pushing.